### DISCUSSION TITLE
can add Prrobot, Skitch, snapbug,Stampeder,Tripod Hopper,Fast Walker,Charger (Xen creature),Unidentified Quiver Creature

### DISCUSSION THREAD

**Discussion by halflifefan123** [VJ-HLR-Developers/Half-Life-Resurgence#40] [Ideas]
can add Prrobot, Skitch, snapbug,Stampeder,Tripod Hopper,Fast Walker,Charger (Xen creature),Unidentified Quiver Creature

**Comment by justwatchinghere** [comment]
so it seems that not all of theses have models so idk if they could do all but i would love to see these
